src/openrct2-ui/windows/Guest.cpp

```diff
@@ -2067,6 +2067,11 @@ void window_guest_debug_paint(rct_window* w, rct_drawpixelinfo* dpi)
     auto peep = GET_PEEP(w->number);
     auto x = w->x + window_guest_debug_widgets[WIDX_PAGE_BACKGROUND].left + 4;
     auto y = w->y + window_guest_debug_widgets[WIDX_PAGE_BACKGROUND].top + 4;
+    {
```

Comment:
what's the purpose of this scope?

Reply:
Fairly sure he's copied the scope from below. There was an `int32_t args[]` here before, I believe. I'm fine with the scope staying, even if it serves no purpose any more — it's stylistically consistent with the rest of the function.

Reply:
ok

Reply:
there used to be a variable created in it, but it was changed to used `set_format_arg`. i left it in because:
1. someone might need to create a variable in it later, as with all the rest of the scopes in this function
2. all the rest of the instances of printing a line of text inside this function are inside a scope so continuing that trend makes the code easier to read

i can take it out if it causes issues

Reply:
edit: didnt see aaron's reply till after i posted mine. he's exactly right